modules/admin-ui/src/main/java/org/opencastproject/adminui/endpoint/AbstractEventEndpoint.java

```diff
@@ -843,12 +844,37 @@ public Response applyAclToEvent(@PathParam("eventId") String eventId, @FormParam
 
       Source eventSource = getIndexService().getEventSource(optEvent.get());
       if (eventSource == Source.ARCHIVE) {
-        if (getAclService().applyAclToEpisode(eventId, accessControlList)) {
+        Opt<MediaPackage> mediaPackage = getAssetManager().getMediaPackage(eventId);
+        Option<AccessControlList> aclOpt = Option.option(accessControlList);
+        // the episode service is the source of authority for the retrieval of media packages
+        if (mediaPackage.isSome()) {
+          MediaPackage episodeSvcMp = mediaPackage.get();
```

Comment:
what does eipisdodeSvcMP stands for? its better than epM or so, but a little longer would make it more readable.

Reply:
As with almost all of the stylistic comments, I didn't actually write this code, I just moved it into a different spot. I agree that it's kinda ugly, but fixing it is not in scope for me right now.